packages/scheduler/src/forks/SchedulerDOM.js

```diff
@@ -219,12 +219,16 @@ function workLoop(hasTimeRemaining, initialTime) {
       currentTask.callback = null;
       currentPriorityLevel = currentTask.priorityLevel;
       const didUserCallbackTimeout = currentTask.expirationTime <= currentTime;
-      markTaskRun(currentTask, currentTime);
+      if (enableProfiling) {
+        markTaskRun(currentTask, currentTime);
```

Comment:
Calls to `markTaskRun` are no-ops when `enableProfiling` is not enabled so let's just remove them.